src/components/settings/SettingsSSOTab/NewSSOConfigForm.jsx

```diff
@@ -0,0 +1,35 @@
+import { Form } from '@edx/paragon';
+import React, { useState } from 'react';
+
+const NewSSOConfigForm = () => {
+  const [metadataMethod, setMetadataMethod] = useState('url'); // vs 'fileupload' in future
+  const [metadataURL, setMetadataURL] = useState('');
+  const handleMetadataUpdate = (event) => setMetadataMethod(event.target.value);
+  const handleMetadataURLChange = (event) => setMetadataURL(event.target.value);
+  const TITLE = 'First, select the way to provide your Identity Provider Metadata and fill out the corresponding fields. ';
+  return (
+    <div>
+      {TITLE}
+
+      <div className="sso-create-form mt-4.5 d-flex">
```

Comment:
totally s hould be, but yeah when I implement the next ticket, forms, this will be include dthere. For this ticket I just placed the initial state of the form, but the form is hardly usable :)